recipes/jobtastic/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "jobtastic" %}
+{% set version = "0.3.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "da49cd83756d6899f2f7b57b1c65b3e3c2a5c3ce7b9fc296aab9ebaa10ec83cc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/PolicyStat/{{ name }}/archive/v{{ version }}.tar.gz
+  {{hash_type}}: {{ hash_val }}
+
+build:
+  skip: true  # [not py27]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - psutil >=3.0,<4
+    - celery >=2.5,<4
```

Comment:
Are these 2 needed at build time or run time only? If the latter, can we please drop them from the build requirements?